api/cpp/esp-idf/slint/include/slint-esp.h

```diff
@@ -37,10 +37,11 @@ struct SlintPlatformConfiguration
     /// The size of the screen in pixels.
     slint::PhysicalSize size;
     /// The handle to the display as previously initialized by `bsp_display_new` or
-    /// `esp_lcd_panel_init`.
-    esp_lcd_panel_handle_t panel = nullptr;
-    /// The touch screen handle, if the device is equipped with a touch screen.
-    esp_lcd_touch_handle_t touch = nullptr;
+    /// `esp_lcd_panel_init`. Must be set to a valid, non-null esp_lcd_panel_handle_t.
+    esp_lcd_panel_handle_t panel_handle = nullptr;
```

Comment:
maybe we shouldn't have = nullptr then?
on the other hand, this doesn't prevent user to not set it and then it would be uninitialized memory

Reply:
Yep, that's exactly the scenario I was worried about. I'd rather we see a crash (or panic) than undefined memory :)